autoload/airline/extensions/dirvish.vim

```diff
@@ -22,10 +22,7 @@ function! airline#extensions#dirvish#apply(...) abort
       \ ? '%{airline#extensions#branch#get_head()}'
       \ : ''
 
-    let w:airline_section_c =
-      \   '%{join(split((execute("lcd"))))}'
-      \ . s:spc
-      \ . get(w:, 'airline_section_c', g:airline_section_c)
+    let w:airline_section_c = '%{expand("%:h")}'
```

Comment:
I think I should write this.

```viml
let w:airline_section_c = '%{b:dirvish._dir}'
```

Reply:
That makes sense to do. I will include it in the next commit I make.